### PR TITLE
Introduce using statements and format codes

### PR DESCRIPTION
**Pull request by NextTurn**
Contributes to dotnet/dotnet-api-docs#1491

### CODE REVIEW

File: snippets/csharp/VS_Snippets_CLR_Classic/classic NotifyFilters Example/CS/source.cs [hunk 1]
```diff
@@ -5,62 +5,59 @@
...
         // Create a new FileSystemWatcher and set its properties.
-        FileSystemWatcher watcher = new FileSystemWatcher();
-        watcher.Path = args[1];
-        /* Watch for changes in LastAccess and LastWrite times, and
-           the renaming of files or directories. */
-        watcher.NotifyFilter = NotifyFilters.LastAccess | NotifyFilters.LastWrite
-           | NotifyFilters.FileName | NotifyFilters.DirectoryName;
-        // Only watch text files.
-        watcher.Filter = "*.txt";
+        using (FileSystemWatcher watcher = new FileSystemWatcher())
```
Comment (rpetrusha) [inline snippets/csharp/VS_Snippets_CLR_Classic/classic NotifyFilters Example/CS/source.cs]: The corresponding VB example is ~\samples\snippets\visualbasic\VS_Snippets_CLR_Classic\classic NotifyFilters Example\VB\source.vb

File: snippets/csharp/VS_Snippets_CLR/ProcessVerbs_Diagnostics/CS/source.cs [hunk 2]
```diff
@@ -59,14 +59,16 @@ static void Main()
                 }
                 // </Snippet4>
 
-                var newProcess = new Process();
-                newProcess.StartInfo = startInfo;
                 try
                 {
-                    newProcess.Start();
+                    using (var newProcess = new Process())
```
Comment (NextTurn) [inline snippets/csharp/VS_Snippets_CLR/ProcessVerbs_Diagnostics/CS/source.cs]: I'm not sure which is better here.

Comment (rpetrusha) [reply snippets/csharp/VS_Snippets_CLR/ProcessVerbs_Diagnostics/CS/source.cs]: I think that this is fine.

**Comment by rpetrusha** [comment]
Thanks for modifying the VB code as well, @NextTurn. I'll merge your PR now.

**Pull request merged**